### ISSUE TITLE
Predefined icon packs are forgotten every time vault is opened

### ISSUE THREAD

**Issue by TomBombTom** [FlorianWoelki/obsidian-iconize#605]
**Describe the bug**
The plugin doesn't seem to recognise or remember the predefined icon packs I have downloaded. They are still present in the /.obsidian/icons folder, but the plugin won't see them, meaning I need to re-download them. In the console, an error is printed (refer to screenshot attached)

**To Reproduce**
1. Download/add any predefined icon packs from the iconize settings.
2. Close vault
3. Reopen vault
4. Check list of icon packs
5. Check /.obsidian/icons, and see that all icon pack zips are still there

**Expected behavior**
Icon packs should still be recognised after closing and opening the vault.

**Screenshots**
<img width="742" alt="Screenshot 2024-11-02 at 10 18 49 am" src="https://github.com/user-attachments/assets/4e4637e6-42ab-46cd-aab1-d00785dada20">

**Additional context**
Obsidian 1.7.4
MacOS 15.1
2020 M1 Macbook Air

**Comment by kylek29** [comment]
Yes, I came across this debugging some performance issues after 1.7.4. I noticed it also generates redundant "INFO: Loaded icon pack 'icon-brew' ..." (used icon-brew just for example) in the console as well as on the Iconize settings page (in this example, Icon-Brew would be listed twice).

**Comment by TomBombTom** [comment]
@FlorianWoelki Not sure if you have seen this, but its quite annoying having to always re-add the packs (despite them still being in the folder)
